packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1279,6 +1279,36 @@ export class NotebookRange implements theia.NotebookRange {
 
 }
 
+export class SnippetTextEdit implements theia.SnippetTextEdit {
+    range: Range;
+    snippet: SnippetString;
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    static isSnippetTextEdit(thing: any): thing is SnippetTextEdit {
```

Comment:
Suggestion: We usually try to use `unknown` in place of `any` where appropriate. Even though they use `any` in the original vscode implementation...

You can use the `isObject` utility function to easily convert `thing` to an instance of `SnippetTextEdit`.